### DISCUSSION TITLE
centered caption with figure directive?

### DISCUSSION THREAD

**Discussion by fohrloop** [executablebooks/meta#1029] [Q&A] [answered]
I was reading the documentation at [MyST Markdown Guide](https://myst-tools.org/docs/mystjs/quickstart-myst-markdown), and saw that the figure caption is not centered: 

![image](https://github.com/executablebooks/meta/assets/17479683/1f3185a5-20ef-4b56-a5f2-344e0775d5ca)

is this the intended behavior? How would one center the caption?

**Comment by rowanc1** [comment, marked as answer]
There is currently a bug for this here:
https://github.com/executablebooks/myst-theme/issues/106
Hope to get this fixed soon!

**Reply by fohrloop** [reply]
Thanks for the prompt response!
